lib/Mojolicious/Sessions.pm

```diff
@@ -9,6 +9,7 @@ has cookie_name        => 'mojolicious';
 has cookie_path        => '/';
 has default_expiration => 3600;
 has deserialize        => sub { \&_deserialize };
+has partitioned        => 0;
```

Comment:
How is this different from the secure flag?

Reply:
Partitioned is used to tell browsers that cookies should be stored in a separate cookie-jar, hidden from other TLDs. You can see the RFC amendment explanation here: https://github.com/privacycg/CHIPS

Reply:
Not what i mean. Why does it have to be initialised with 0, unlike other flags?

Reply:
You're correct, I had it as `1` to begin with, but then I assumed we'd probably not want to default it on. I'll change it to be in the other declaration, good catch.